sklearnex/linear_model/tests/test_logreg.py

```diff
@@ -129,5 +129,5 @@ def test_csr(queue, dtype, dims):
 
         assert_allclose(pred, pred_sp)
         assert_allclose(prob, prob_sp)
-        assert_allclose(model.coef_, model_sp.coef_)
-        assert_allclose(model.intercept_, model_sp.intercept_)
+        assert_allclose(model.coef_, model_sp.coef_, atol=1e-6)

```

Comment:
you can increase tolerance up to 1e-5 to avoid sporadic failures

Reply:
Agreed, or even 1e-4. I see in the nightly that max relative difference was 7.3e-5

Reply:
I believe rtol is a better metric since its independent of data

Reply:
done